docs/source/user_guide/connector/connector_authorization.rst

```diff
@@ -0,0 +1,185 @@
+The following property should be included below the ``method`` property in your configuration file::
+
+    "authorization": {
+        "type": "OAuth2",
+        "grantType": "ClientCredentials",
+        "tokenServerUrl": "<OAuth 2.0 token URL>"
+    },
+
+You must replace the ``<OAuth 2.0 token URL"`` string with the OAuth 2.0 token URL defined by the Web API. Review the Web API documentation to identify the exact URL.
+
+For example, see the `Twitter API - Tweets endpoint <https://github.com/sfu-db/DataConnectorConfigs/blob/develop/twitter/tweets.json>`_, configuration below::
+
+    "authorization": {
+        "type": "OAuth2",
+        "grantType": "ClientCredentials",
+        "tokenServerUrl": "https://api.twitter.com/oauth2/token"
+    },
+
+Before executing any query, you must get your Web API ``Client ID`` and ``Client Secret`` information. Once obtained, you can pass these values as parameters in the ``connect`` function and then execute the ``query`` method to retrieve the desired data:::
+
+    import asyncio
+    from dataprep.connector import connect
+    twitter_connector = connect("twitter", _auth={"client_id":twitter_client_id, "client_secret":twitter_client_secret})
+    df = await twitter_connector.query("tweets", q="data science")
+
+Internally, Connector will take both ``Client ID`` and ``Client Secret`` values and execute the OAuth 2.0 - Client Credentials grant process on your behalf.
+
+
+Authorization Code grant
+--------------------------
+In your configuration file, you have to specify that you'll use the OAuth 2.0 authorization type - Authorization Code grant.
+The following property should be included below the ``method`` property in your configuration file::
+
+    "authorization": {
+        "type": "OAuth2",
+        "grantType": "AuthorizationCode",
+        "tokenServerUrl": "<OAuth 2.0 token URL>"
+    },
+
+You must replace the ``<OAuth 2.0 token URL"`` string with the OAuth 2.0 token URL defined by the Web API. Review the Web API documentation to identify the exact URL.
+
+For example, see the Twitter API - Tweets endpoint - configuration below::
+
+    "authorization": {
+        "type": "OAuth2",
+        "grantType": "AuthorizationCode",
+        "tokenServerUrl": "https://api.twitter.com/oauth2/token"
+    },
+
+Before executing any query, you must get your Web API ``Client ID`` and ``Client Secret`` information. Once obtained, you can pass these values as parameters in the ``connect`` function and then execute the ``query`` method to retrieve the desired data:::
+
+    import asyncio
+    from dataprep.connector import connect
+    twitter_connector = connect("twitter", _auth={"client_id":twitter_client_id, "client_secret":twitter_client_secret})
+    df = await twitter_connector.query("tweets", q="data science")
+
+Internally, Connector will take both ``Client ID`` and ``Client Secret`` values and execute the OAuth 2.0 - Authorization Code grant process on your behalf.
+
+That's all for now.
```

Comment:
Let's delete the content after 'That's all for now.' (including 'That's all for now.')

Reply:
Done.